test/OrleansTestKit.Tests/Grains/BasicGrains.cs

```diff
@@ -10,11 +10,23 @@ public sealed class GuidKeyGrain : Grain, IGuidKeyGrain
         public Task<Guid> GetKey() => Task.FromResult(this.GetPrimaryKey());
     }
 
+    public sealed class GuidCompoundKeyGrain : Grain, IGuidCompoundKeyGrain
+    {
+        private string _keyExt;
+        public Task<(Guid, string)> GetKey() => Task.FromResult((this.GetPrimaryKey(out _keyExt), _keyExt));
+    }
+
     public sealed class IntegerKeyGrain : Grain, IIntegerKeyGrain
     {
         public Task<long> GetKey() => Task.FromResult(this.GetPrimaryKeyLong());
     }
 
+    public sealed class IntegerCompoundKeyGrain : Grain, IIntegerCompoundKeyGrain
+    {
+        private string _keyExt;
+        public Task<(long, string)> GetKey() => Task.FromResult((this.GetPrimaryKeyLong(out _keyExt), _keyExt));
```

Comment:
As a side note, ValueTuples are not currently supported in Orleans. This is find to use in the TestKit since we dont use the codegen/serialization from Orleans.